Settings: FAQ button

rperez89 opened the pull request:
![Screen Shot 2020-04-14 at 5 05 16 PM](https://user-images.githubusercontent.com/11763623/79269042-304dd480-7e72-11ea-89c1-826ad91047e5.png)

fabriziovigevani commented on src/components/GlobalPreferences/GlobalPreferencesButton.js:
```diff
@@ -84,17 +85,34 @@ function GlobalPreferencesButton({ onOpen }) {
             label="Network"
           />
+          <Item
+            onClick={handleItemClick('network')}
```
I think this one should be removed ?

rperez89 replied:
ahh yes good catch!

rperez89 commented:
> I was thinking that maybe we can throw the `iconFirst`prop and differentiate whether we need to add the external icon or not by the `href`. And still tell if we should add the icon first just by `icon`
> 
> What do you think?

mmm not sure about this one, we might  have another preferences button that the icon goes to the right and is not an external one

sohkai commented on src/components/GlobalPreferences/GlobalPreferencesButton.js:
```diff
@@ -84,17 +85,33 @@ function GlobalPreferencesButton({ onOpen }) {
 }
 
-function Item({ icon, label, onClick }) {
+function Item({ icon, iconFirst, label, onClick, href }) {
```
Rather than exposing an API with `iconFirst`, what do you think about exposing `children` instead?

We've used this pattern a few times; with that you can opt for the "easy" pre-defined layout, or customize the internals completely by using children.

sohkai commented on src/components/GlobalPreferences/GlobalPreferencesButton.js:
```diff
@@ -84,17 +85,33 @@ function GlobalPreferencesButton({ onOpen }) {
   const theme = useTheme()
+  let Icon
+
+  switch (icon) {
```
Let's update the old API as we find new edge cases :).

Rather than doing this, let's force users to pass down the node they'd like to render in the icon slot (e.g. users would pass down `IconExternal` or `<img src={icon} />`).

sohkai commented:
@rperez89 I've added a few commits to fix the spacing and revert the `Item`'s API to be the old one.

If you look at the ["removed whitespace" diff](https://github.com/aragon/court-dashboard/pull/279/files?diff=unified&w=1) you'll see that we didn't have to change very much at all to add this "exception"!

rperez89 commented:
> @rperez89 I've added a few commits to fix the spacing and revert the `Item`'s API to be the old one.
> 
> If you look at the ["removed whitespace" diff](https://github.com/aragon/court-dashboard/pull/279/files?diff=unified&w=1) you'll see that we didn't have to change very much at all to add this "exception"!

Hi! not sure why we need that margin? and not sure why we reverted the Icon to the old one?

rperez89 commented:
> @rperez89 I've added a few commits to fix the spacing and revert the `Item`'s API to be the old one.
> 
> If you look at the ["removed whitespace" diff](https://github.com/aragon/court-dashboard/pull/279/files?diff=unified&w=1) you'll see that we didn't have to change very much at all to add this "exception"!

yeah! agree. Was a simple but nice exercise to do
